src/time_span.rs

```diff
@@ -0,0 +1,49 @@
+use crate::*;
+
+/// Represents a time interval as a signed 64-bit integer value.
+///
+/// TimeSpan represents the WinRT [TimeSpan](https://docs.microsoft.com/en-us/uwp/api/Windows.Foundation.TimeSpan)
+/// struct and provides convertibility with `std::time::Duration`.
+#[repr(C)]
+#[derive(Clone, Default, Debug, PartialEq)]
```

Comment:
Any reason for this not to be copy?

Reply:
Good catch - I just copied the default we generate for WinRT structs. But for TimeSpan Copy makes sense.